content/en/blog/2023/otel-arrow/index.md

```diff
@@ -7,7 +7,7 @@ date: 2023-11-14
 author: >-
   [Laurent Quérel](https://github.com/lquerel) (F5), [Joshua
   MacDonald](https://github.com/jmacd) (ServiceNow)
-cSpell:ignore: Quérel SIMD Zstandard zstd
+cSpell:ignore: SIMD Zstandard zstd
```

Comment:
Please revert this change on the 2023 blog post. Thanks!

Reply:
Oops I didn't mean this. Thanks.